app/pages/shipyard/ShipyardOccupationList1Page.scala

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package pages.shipyard
+
+import pages.QuestionPage
+import play.api.libs.json.JsPath
+
+case object ShipyardOccupationList1Page extends QuestionPage[Boolean] {
+
+  override def path: JsPath = JsPath \ toString
```

Comment:
We store Industries in a sub directory called "Industry", have a look at other industry pages for how this is done